### PR TITLE
OCM-7146 | fix: qe review comments for 1.6.2-prerelease.1

### PR DESCRIPTION
**Pull request by gdbranco**
[OCM-8088](https://issues.redhat.com//browse/OCM-8088) - Link to submodules in documentation return 404
[OCM-8089](https://issues.redhat.com//browse/OCM-8089) - Document `path` field format
[OCM-8090](https://issues.redhat.com//browse/OCM-8090) - Main module should provide `aws_billing_account_id` variable
[OCM-8092](https://issues.redhat.com//browse/OCM-8092) | feat: include example usage and registry source

### CODE REVIEW

File: examples/rosa-hcp-public/README.md [hunk 1]
```diff
@@ -8,6 +8,63 @@ This example includes:
...
+locals {
+  account_role_prefix  = "${var.cluster_name}-account"
+  operator_role_prefix = "${var.cluster_name}-operator"
+}
+
+# ############################
+# Cluster
+############################
+module "hcp" {
+  source = "terraform-redhat/rosa-hcp/rhcs"
```
Comment (radtriste) [inline examples/rosa-hcp-public/README.md:23]: no version ?

File: main.tf [hunk 2]
```diff
@@ -14,7 +14,8 @@ locals {
 ##############################################################
 
 module "account_iam_resources" {
-  source = "./modules/account-iam-resources"
+  source = "terraform-redhat/rosa-hcp/rhcs//modules/account-iam-resources"
```
Comment (radtriste) [inline main.tf:17]: This is merged on `main` branch, don't we want to keep there the relative paths ?

File: README.md [hunk 3]
```diff
@@ -5,17 +5,39 @@ Terraform module which creates ROSA HCP cluster
 
 This module serves as a comprehensive solution for deploying, configuring and managing Red Hat OpenShift on AWS (ROSA) Hosted Control Plane (HCP) clusters within your AWS environment. With a focus on simplicity and efficiency, this module streamlines the process of setting up and maintaining ROSA HCP clusters, enabling users to use the power of OpenShift on AWS infrastructure effortlessly.
 
+## Example Usage
+
+```
+module "hcp" {
+  source = "terraform-redhat/rosa-hcp/rhcs"
+
+  cluster_name           = var.cluster_name
```
Comment (radtriste) [inline README.md:14]: variables definition ?
I think the example should contain a fully defined TF

File: modules/account-iam-resources/README.md [hunk 4]
```diff
@@ -8,6 +8,24 @@ These IAM resources can be created once and used across multiple ROSA HCP cluste
 
 For more information, see [About IAM resources for ROSA clusters that use STS](https://docs.openshift.com/rosa/rosa_architecture/rosa-sts-about-iam-resources.html#rosa-sts-about-iam-resources) in the ROSA documentation.
 
+## Example Usage
+
+```
+locals {
+  account_role_prefix  = "${var.cluster_name}-account"
```
Comment (radtriste) [inline modules/account-iam-resources/README.md:15]: variables ?

**Review by radtriste — COMMENTED** [review]
I know the PR has been merged but please have a look to my comments :)

File: main.tf (hunk 2 above)
Comment (gdbranco) [reply main.tf:17]: This generates registry docs which end up pointing to the wrong place

Comment (gdbranco) [reply main.tf:17]: I'd rather also keep it local, but idk if it is proper considering the registry

Comment (radtriste) [reply main.tf:17]: `main` branch should keep it local
We should change this to absolute link only when performing a release IMO

**Pull request merged**